ui/app/pages/swaps/actionable-message/index.scss

```diff
@@ -83,7 +83,6 @@
 
     .actionable-message__action {
       font-weight: normal;
-      cursor: pointer;
```

Comment:
The other `cursor: pointer` (also on `actionable-message__action`, declared earlier) is now unnecessary as well

Reply:
good point, removed in 3a38844e7